pkg/ccl/backupccl/backup_processor.go

```diff
@@ -337,8 +330,8 @@ func runBackupProcessor(
 				// original span, and we must update the existing progress object.
```

Comment:
This comment doesn't seem relevant anymore with the map gone.

Reply:
good catch -- i'll fix on master but leave the backdoor the original commit for now.